typings/single-spa.d.ts

```diff
@@ -4,16 +4,37 @@ declare module "single-spa" {
   	[p in keyof T]: Array<T[p]>
   };
 
-  type AppProps = {
-  	name: string,
-  	mountParcel(): any,
-	singleSpa: any,
+  export type AppProps = {
```

Comment:
Used here: https://github.com/CanopyTax/single-spa-angular/pull/87/files#diff-987cb5d8ba0f22ac151cf1bcade73452R2

idk of a way to be able to use this type without exporting it.